package.json

```diff
@@ -17,20 +17,21 @@
   },
   "license": "MIT",
   "author": "Keith Cirkel <npm@keithcirkel.co.uk> (http://keithcirkel.co.uk/)",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "bin": "cli.js",
   "files": [
     "index.js",
     "index.d.ts",
     "cli.js"
   ],
-  "main": "index.js",
-  "types": "index.d.ts",
-  "bin": "cli.js",
   "scripts": {
     "lint": "eslint .",
     "semantic-release": "semantic-release",
     "sort-package-json": "node cli.js package.json --check",
     "test": "ava",
-    "test-coverage": "nyc ava"
+    "test-coverage": "nyc ava",
+    "update-snapshot": "ava -u && node cli.js package.json"
```

Comment:
I dont think we need this, its on `ava` docs

Reply:
but this also help when update sort order, can more easy update package.json and snapshot at same time

Reply:
I like this. I’d probably forget `-u` and this is a nice reminder.